lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -623,4 +646,38 @@ class AwsCompileFunctions {
   }
 }
 
+function addFileContentsToHashes(filePath, hashes) {
+  return new BbPromise((cb, reject) => {
```

Comment:
Let's use native `Promise`, and for consistency let's use `resolve` name instead `cb`